runtime/src/window/action.rs

```diff
@@ -81,6 +81,11 @@ pub enum Action<T> {
     GainFocus(Id),
     /// Change the window [`Level`].
     ChangeLevel(Id, Level),
+    /// Show window menu at cursor position.
+    ///
+    /// ## Platform-specific
+    /// Android / iOS / macOS / Orbital / Wayland / Web / X11: Unsupported.
+    ShowWindowMenu(Id),
```

Comment:
You said, it is supported on Wayland?

Reply:
Ah, I copied that from winit. It looks like the comment there wasn't updated when the Wayland implementation was added.